blog/2023-10-29-service-exceptions-and-translations.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Should we change the date of the blog in the filename to today's date? It's a bit old now.

Reply:
Good point, I'll do that